src/sentry/static/sentry/app/components/forms/selectControl.jsx

```diff
@@ -164,6 +164,33 @@ const defaultStyles = {
   }),
 };
 
+const getFieldLabelStyle = label => ({
+  ':before': {
+    content: `"${label}"`,
+    color: theme.gray300,
```

Comment:
This should be ok for dark mode atm, since we are not (yet) changing colors of the select component in dark mode.